consensus/hotstuff/votecollector/interfaces.go

```diff
@@ -0,0 +1,79 @@
+package votecollector
+
+import (
+	"github.com/onflow/flow-go/crypto"
+	"github.com/onflow/flow-go/model/flow"
+)
+
+// SigType is the aggregable signature type.
+type SigType int
+
+// There are two signatures are aggregable, one is the normal staking signature,
```

Comment:
This whole file doesn't make sense I think since we have other interfaces for aggregators